.github/workflows/jdt-aspectj-deploy.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#apache-maven-with-a-settings-path
+
+name: Deploy AspectJ JDT Core
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 14
+        uses: actions/setup-java@v2
+        with:
+          java-version: '14'
```

Comment:
When I finally saw the pom.xml, I see it is set to source/compliance 8, so maybe that is fine. Those constraints can run on 14.  I suppose the only thing that might slip through if the build isn't run on 8 is if a system class from a later version of Java is referenced and it isn't built with strict compliance rules for that (not sure if it is).  Happy to go with 14 for now, but we must confirm the resultant AspectJ at the end of all this runs on older versions of Java.

Reply:
Like I told you via e-mail in more detail on 2021-03-31:

> The `java:javax15api` system dependency points to `org.eclipse.jdt.compiler.apt/lib/javax15api.jar`, otherwise we cannot compile the APT files because they refer to it. Because `javax15api.jar` contains Java 14 class files, I have to use the forked AspectJ Maven plugin I mentioned above and and compile with JDK 14+, but I can use source/target 1.8.

So yes, we need Java 14+.